feat: added tailscale option

DaRandomCube opened the pull request:
I hope that's good enough

Didn't know where to actually put the additions so i decided to put it in common.just considering that
1. Tailscale is a utility
2. Similar stuff are in common.just, VPNs for example

DaRandomCube committed:
added tailscale options (idk if that's the right justfile)

DaRandomCube committed:
moved the tailscale option to be in `ujust install-vpn`

DaRandomCube committed:
added the tailscale repo file

DaRandomCube committed:
Merge branch 'secureblue:live' into live

RoyalOughtness commented on files/justfiles/common/utilities.just:
```diff
@@ -327,10 +328,26 @@ install-vpn:
+
+            "tailscale")
+                PACKAGE_NAMES="tailscale"
+                REPO_NAME="tailscale"
```
the repo name doesn't match the name in the file

RoyalOughtness commented:
please screenshot the ujust output when run locally

DaRandomCube commented:
here it is (no screenshot because it's long)
```
qemu@secureblue:/var/home/qemu/Downloads$ just -f ./utilities.just install-vpn
Using a VPN is currently incompatible with global DNS or local DNSSEC resolution.
If you are using Unbound, it will be disabled in favour of systemd-resolved.
Do you want to continue? [y/N] y
 
NOTE: You can also install wireguard profiles from your provider directly into your DE, if you prefer.
Do you want to continue? [y/N] y
 
Please choose your desired provider
You selected tailscale
 
Detected DNS resolver as Unbound.
Setting DNS resolver to systemd-resolved...
 
Starting installation...
Installing Tailscale VPN...
^[]11;rgb:2323/2626/2727^[\^[]11;rgb:2323/2626/2727^[\⠁ Checking out tree 581f3d5...                                                                                                                                                        Checking out tree 581f3d5... done
Enabled rpm-md repositories: fedora-multimedia updates fedora secureblue copr:copr.fedorainfracloud.org:secureblue:branding copr:copr.fedorainfracloud.org:secureblue:bubblejail copr:copr.fedorainfracloud.org:secureblue:crane copr:copr.fedorainfracloud.org:secureblue:hardened_malloc copr:copr.fedorainfracloud.org:secureblue:no_rlimit_as copr:copr.fedorainfracloud.org:secureblue:run0edit copr:copr.fedorainfracloud.org:secureblue:slsa-verifier tailscale-stable updates-archive
Updating metadata for 'fedora-multimedia'... done
Updating metadata for 'updates'... done
⠉ Updating metadata for 'fedora'  95% [███████████████████░] (0s)
Updating metadata for 'fedora'... done
Updating metadata for 'secureblue'... done
Updating metadata for 'copr:copr.fedorainfracloud.org:secureblue:branding'... done
Updating metadata for 'copr:copr.fedorainfracloud.org:secureblue:bubblejail'... done
Updating metadata for 'copr:copr.fedorainfracloud.org:secureblue:crane'... done
Updating metadata for 'copr:copr.fedorainfracloud.org:secureblue:hardened_malloc'... done
Updating metadata for 'copr:copr.fedorainfracloud.org:secureblue:no_rlimit_as'... done
Updating metadata for 'copr:copr.fedorainfracloud.org:secureblue:run0edit'... done
Updating metadata for 'copr:copr.fedorainfracloud.org:secureblue:slsa-verifier'... done
Updating metadata for 'tailscale-stable'... done
Updating metadata for 'updates-archive'... done
Importing rpm-md... done
rpm-md repo 'fedora-multimedia'; generated: 2025-12-01T09:42:00Z solvables: 417
rpm-md repo 'updates'; generated: 2025-12-02T00:44:46Z solvables: 12836
rpm-md repo 'fedora'; generated: 2025-10-23T03:37:20Z solvables: 77664
rpm-md repo 'secureblue'; generated: 2025-11-18T02:01:58Z solvables: 146
rpm-md repo 'copr:copr.fedorainfracloud.org:secureblue:branding'; generated: 2025-11-20T03:32:25Z solvables: 4
rpm-md repo 'copr:copr.fedorainfracloud.org:secureblue:bubblejail'; generated: 2025-11-09T03:50:51Z solvables: 2
rpm-md repo 'copr:copr.fedorainfracloud.org:secureblue:crane'; generated: 2025-11-27T03:39:00Z solvables: 4
rpm-md repo 'copr:copr.fedorainfracloud.org:secureblue:hardened_malloc'; generated: 2025-10-16T17:00:20Z solvables: 4
rpm-md repo 'copr:copr.fedorainfracloud.org:secureblue:no_rlimit_as'; generated: 2025-11-03T03:28:37Z solvables: 4
rpm-md repo 'copr:copr.fedorainfracloud.org:secureblue:run0edit'; generated: 2025-10-30T03:09:51Z solvables: 2
rpm-md repo 'copr:copr.fedorainfracloud.org:secureblue:slsa-verifier'; generated: 2025-11-13T03:48:53Z solvables: 4
rpm-md repo 'tailscale-stable'; generated: 2025-11-26T22:00:57Z solvables: 133
rpm-md repo 'updates-archive'; generated: 2025-12-02T01:12:52Z solvables: 12988
Resolving dependencies... done
Will download: 1 package (34.0 MB)
Downloading from 'tailscale-stable'... done
Importing packages... done
Checking out packages... done
Running systemd-sysusers... done
Running pre scripts... done
Running post scripts... done
Running posttrans scripts... done
Writing rpmdb... done
Writing OSTree commit... done
Staging deployment... done
Added:
  tailscale-1.90.9-1.x86_64
Changes queued for next boot. Run "systemctl reboot" to start a reboot
Successfully installed Tailscale VPN, please execute the command "run0 systemctl enable --now tailscaled && run0 tailscale up".
Please reboot your device to fully apply the changes.
qemu@secureblue:/var/home/qemu/Downloads$
```

@RoyalOughtness you most likely forgot adding the repo file, i added it a while ago

RoyalOughtness committed:
Merge branch 'live' into live

DaRandomCube committed:
remove a white space because it causes an error with the linter

RoyalOughtness committed:
Update utilities.just
